scanner/indexer/indexer.go

```diff
@@ -26,12 +27,23 @@ import (
 	"github.com/stackrox/rox/scanner/internal/version"
 )
 
+// ReportGetter can get index reports from an Indexer.
+type ReportGetter interface {
+	GetIndexReport(context.Context, string) (*claircore.IndexReport, bool, error)
+}
+
+// RemoteIndexer represents the interface offered by remote indexers.
```

Comment:
Should this be moved closer to where it's used (indexer/remote.go)?

Reply:
7b472c4ee6..a70764c524

Reply:
Please resolve if no more comments.